springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/annotation/AsyncAnnotationChannelsScanner.java

```diff
@@ -94,13 +94,7 @@ private Stream<MethodAndAnnotation<A>> getAnnotatedMethods(Class<?> type) {
     }
 
     private boolean isInvalidChannel(Map.Entry<String, ChannelObject> entry) {
-        //        Operation publish = entry.getValue().getPublish();
-        //        boolean publishBindingExists = publish != null && publish.getBindings() != null;
-        //
-        //        Operation subscribe = entry.getValue().getSubscribe();
-        //        boolean subscribeBindingExists = subscribe != null && subscribe.getBindings() != null;
-
-        boolean allNonNull = entry.getKey() != null; // && (publishBindingExists || subscribeBindingExists); FIXME
+        boolean allNonNull = entry.getKey() != null;
```

Comment:
I believe a map key can never be null be design.
-> remove this method?

Reply:
Yes, I think that would be the best. If in the future we find out that some conditions are generating broken instances of Channel or Operation, we can add back the filter